docs/it/tut-polishing-jettycat.md

```diff
@@ -296,4 +302,7 @@ Il menu principale ora avrà un cielo animato e il cielo nel livello di gioco pr
 
 Il gioco è rifinito e sembra fantastico, evviva! È ora di leggere altri tutorial o di creare un nuovo gioco da zero!
 
-Buona codifica!
+Ricorda che puoi pubblicare i tuoi giochi su **itch.io** o piattaforme di giochi on line simili. È facile, [leggi le istruzioni nella guida](deployment-itch-io.html).
```

Comment:
Let's not deviate much from the english docs.

Reply:
ok